atlas/templates/.gitignore.gotmpl

```diff
@@ -1 +1,4 @@
 bin
+.idea
```

Comment:
good to have this ignored in your global `.gitignore` instead of making it a part of specific repositories.